compiler/plugins/target/AMD-AIE/aie/AMDAIECreatePathFindFlows.cpp

```diff
@@ -89,6 +89,23 @@ ConnectOp getOrCreateConnect(OpBuilder &builder, Operation *parentOp,
                                    srcChannel, destBundle, destChannel);
 }
 
+PacketRulesOp getOrCreatePacketRules(OpBuilder &builder, SwitchboxOp &swboxOp,
+                                     StrmSwPortType bundle, int channel) {
+  Block &b = swboxOp.getConnections().front();
```

Comment:
As you're changing the insertion point in this function, add an `InsertionGuard` at the beginning.